packages/@aws-cdk/aws-appmesh/lib/http-route-path-match.ts

```diff
@@ -0,0 +1,90 @@
+import { CfnRoute } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Path and Prefix properties for HTTP route match.
+ */
+export interface HttpRoutePathMatchConfig {
+  /**
+   * Route CFN configuration for path match.
+   *
+   * @default - no path match.
+   */
+  readonly pathMatch?: CfnRoute.HttpPathMatchProperty;
```

Comment:
How about `nonPrefixPathMatch`?

I just find the combo of `pathMatch` and `prefixMatch` confusing.

Reply:
I just thought of the perfect name!

`wholePathMatch`.

It's the perfect antonym to `prefixPathMatch`.

Can you use that pair (`prefixPathMatch` and `wholePathMatch`) everywhere (here, in `HttpGatewayRouteSpec`, in `HttpRouteSpec`, etc.)?

Reply:
I gotcha!

Reply:
@skinny85 - also to reduce confusion, I renamed `pathMatchConfig` variable to `matchConfig`.

Reply:
Sorry - where did you do that renaming?